src/Admin/Admin.jsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import {Button, Container, Form} from "react-bootstrap";
+import {withRouter} from "react-router-dom";
+import {applyFeatures, getFeatures} from "../Service/RestService";
+import ReactLoading from "react-loading";
+import LoadingOverlay from "react-loading-overlay";
+
+import './Admin.css'
+
+class Admin extends React.Component {
+
+  state = {
+    data: [],
+    isLoading: false,
+  };
+
+  componentDidMount() {
+    this.setState({isLoading: true});
+    getFeatures()
+    .then((response) => {
+      this.setState({isLoading: false, data: response});
+
+    }).catch(error => {
+      this.setState({isLoading: false});
+      alert(error.message || 'Sorry! Something went wrong. Please try again!')
+    });
+  }
+
+  handleChange = (event) => {
+    const {id} = event.currentTarget;
+    this.getValue(id).enabled = event.target.checked;
+    this.forceUpdate();
```

Comment:
It's not recommended to use `forceUpdate()` method. Probbly some other approach?